jihyeon/controllers/likesController.js

```diff
@@ -0,0 +1,22 @@
+const likesService = require("../services/likesService");
+
+const likes = async (req, res) => {
+  try {
+    const { userId, postId } = req.body;
+
+    const userLikes = await likesService.likes(userId, postId);
+
+    if (userLikes == true) {
+      return res.status(201).json({ message: "likeCreated" });
+    } else if (userLikes == false) {
+      return res.status(201).json({ message: "likeCanceled" });
+    }
```

Comment:
```suggestion
    if (userLikes == true) return res.status(201).json({ message: "likeCreated" });
    
    return res.status(201).json({ message: "likeCanceled" });
```
똑같이 동작하고 보다 읽기 간결합니다! 실제 개발할 때에는 보통 else는 가급적 쓰지 않고 작성하게 됩니다.